contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -181,6 +207,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
         address borrower,
         uint256 repayAmount // solhint-disable-line no-unused-vars
     ) external returns (uint256) {
+        ensureUserAllowed(payer);
```

Comment:
```suggestion
        ensureUserAllowed(payer);
        ensureUserAllowed(borrower);
```

Reply:
We could check both wallets, but theoretically `borrower` will be allowed, because we checked it in the borrow flow. So, maybe it's not needed